Components/Transform/CMakeLists.txt

```diff
@@ -38,6 +38,12 @@ target_link_libraries(${QUICC_CURRENT_COMPONENT_LIB} PUBLIC
   BLAS::BLAS # banded/tridiag solve/mul TODO: remove dep
   )
 
+# Wrappers depends on kk part
```

Comment:
Doesn't that brink kokkos into everyhting?

Reply:
?  the compilation units are separate and the headers are not exposed